### ISSUE TITLE
new contact -> filter accounts issue (500 server error)

### ISSUE THREAD

**Issue by ahmadgolzar** [orocrm/crm#142]
If you create new contact and then try to filter accounts at the bottom, filtering does not work and you'll get a 500 server error:

"An Exception was thrown while handling: Cannot bind datasource parameter "contact", there is no datagrid parameter with path "contact"."

We have fixed it this way:

```diff
--- vendor/oro/crm/src/OroCRM/Bundle/ContactBundle/Resources/views/Contact/update.html.twig
+++ vendor/oro/crm/src/OroCRM/Bundle/ContactBundle/Resources/views/Contact/update.html.twig
@@ -135,7 +135,7 @@
                 'data' : [
                     form_widget(form.appendAccounts, {'id': 'appendAccounts'}),
                     form_widget(form.removeAccounts, {'id': 'removeAccounts'}),
-                    dataGrid.renderGrid(gridName, { contact: form.vars.value.id }, { cssClass: 'inner-grid' })
+                    dataGrid.renderGrid(gridName, { contact: form.vars.value.id ? form.vars.value.id : -1 }, { cssClass: 'inner-grid' })
                 ]
             }
```

**Comment by Grygir** [comment]
Thanks for pointing out the issue and for contributing the solution. It will be fixed in upcoming release.

**Comment by artml** [comment]
This issue has been closed in 1.9.0 release
